Tests/SonarQube.TeamBuild.PreProcessor.Tests/PreProcessorExeTests.cs

```diff
@@ -21,14 +21,16 @@ public class PreProcessorExeTests
         [TestMethod]
         public void PreProc_ErrorCodeReturnedForInvalidCommandLine()
         {
-            CheckExecutionFails("invalid command line");
+            string errorOutput = CheckExecutionFails("invalid_command_line aaa 123");
+
+            Assert.IsTrue(errorOutput.Contains("invalid_command_line"), "Error output did not contain the expected data. Check that the exe failed for the expected reason");
```

Comment:
The test wasn't specific enough i.e. the exe was failing, but not for the expected reason. Added a new check to cover this.